src/test/java/seedu/address/logic/commands/CompleteCommandTest.java

```diff
@@ -36,22 +36,30 @@
 
     private static final LabelMatchesKeywordPredicate PREDICATE_FRIENDS = new LabelMatchesKeywordPredicate("friends");
     private static final LabelMatchesKeywordPredicate PREDICATE_NONSENSE = new LabelMatchesKeywordPredicate(
-        "AOSDIJPQWEOIDJPQWOiodj120349871238493qw");
+            "AOSDIJPQWEOIDJPQWOiodj120349871238493qw");
     private Model model = new ModelManager(getTypicalTaskManager(), new UserPrefs());
     private CommandHistory commandHistory = new CommandHistory();
 
     @Test
     public void execute_validIndexUnfilteredList_success() {
```

Comment:
could you rename this method name? It doesn't follow style, and I can't really understand what it does from the name.